cdap-master/src/main/java/co/cask/cdap/data/runtime/main/MasterServiceMain.java

```diff
@@ -183,8 +185,18 @@
   }
 
   public static void main(final String[] args) throws Exception {
```

Comment:
This will also initiate the DatasetClassRewriter rewrite is that fine or should we change MainClassLoader to accept what specific rewrite needs to be performed ?